src/OpenTelemetry.Instrumentation.Dependencies/Implementation/SqlClientDiagnosticListener.cs

```diff
@@ -104,12 +102,17 @@ public override void OnCustom(string name, Activity activity, object payload)
                                         break;
 
                                     case CommandType.Text:
+                                        activity.AddTag(DatabaseStatementTypeSpanAttributeKey, "Text");
```

Comment:
Since all cases has this code, I would move it outside the switch and change it to a simple if condition, because we only have an specific condition for Text and if we have the command available.

Reply:
@eddynaka sorry I'm not totally following what you're saying. Would you mind sending me a snippet?

Reply:
@CodeBlanch , didn't see that the `StoredPRocedure` had a logic as well besides adding to the tag. so, just forget it :)